src/signer_node/mod.rs

```diff
@@ -118,6 +121,7 @@ pub enum NodeState {
         round_is_done: bool,
     },
     Member {
+        /// Same with Master state's block_key.
```

Comment:
Each block_key is generated randomly in each signer, and so is not the same as the master's one.

Reply:
I was intent the explanation for the block_key is `same` with master's one. I feel it hard to understand. I will change the comment.